index.html

```diff
@@ -5,26 +5,58 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Awesome-books</title>
-    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/style.css" />
   </head>
   <body>
-    <h1>All Awesome Books</h1>
+    <header>
+      <div class="header-div">
+        <h4>Awesome Books</h4>
+        <nav>
+          <ul class="header-list">
+            <li class="item"><a href="#books">List</a></li>
+            <li class="item"><a href="#add-book">Add new</a></li>
+            <li class="item2"><a href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+      </div>
+      <h4 class="date">March 22nd 2023, 1:30:00 am</h4>
+    </header>
+    <section id="books">
+      <h1>All Awesome Books</h1>
+      <div class="books-list" id="books-list">
+        <ul class="books" id="books">
+          <!-- Contant dynamically generated with JavaScript -->
+        </ul>
+      </div>
```

Comment:
- Hi, I did a nice job in implementing the change, but you have not done yet. Can you add a title to the List of books page frame? This will match the design requirements and also will make your App more attractive. 
- Your Design:
![Capture](https://user-images.githubusercontent.com/63972670/227041369-5a99e858-e774-422b-a2df-b2f18fe08120.PNG)
- Original design:
![Capture1](https://user-images.githubusercontent.com/63972670/227041452-0a68c823-05e3-4721-b114-c3461cd89e6a.PNG)